docs/sidebars.js

```diff
@@ -146,6 +146,17 @@ const sidebars = {
         },
       ]
     },
+    {
+      type: 'category',
+      label: 'How-To',
```

Comment:
We now have "Workflows" in Constellation (which also covers day 1/2 actions of Constellation itself), "Guides" in Privatemode, and "How-To" in Contrast.
I like the explicit naming of "How-To" though:-)
Question is what happens to the "Workload deployment" page.
Should we refactor that into its own section and where we have all the "Contrast operation" references or would it become part of "How-to"?

Reply:
> We now have "Workflows" in Constellation (which also covers day 1/2 actions of Constellation itself), "Guides" in Privatemode, and "How-To" in Contrast.
> I like the explicit naming of "How-To" though:-)

I'm open to renaming to "Guides", but not a big fan of "Workflows". 

> Question is what happens to the "Workload deployment" page.

I'd keep the workload deployment guide where it is right now, but move (and expand) some of its content to How-Tos.

Reply:
Sounds good, let's keep How-To, I like it:-)